build_config/pubspec.yaml

```diff
@@ -21,6 +21,8 @@ dev_dependencies:
   test: ^1.6.0
 
 dependency_overrides:
+  # just for testing...
+  yaml: 3.0.0-nullsafety.0
```

Comment:
I would go ahead and remove this now

Reply:
Yup! just making sure we were green first

Reply:
done